includes/Main.php

```diff
@@ -1153,34 +1154,41 @@ public function temporary_files_cleanup( $delete_timestamp = 0 ) {
 			'.htaccess',
 			'log.htm',
 		) );
-		$folders_level    = apply_filters( 'wpo_wcpdf_cleanup_folders_level', 3 );
+		$folders_level    = apply_filters_deprecated( 'wpo_wcpdf_cleanup_folders_level', array( 3 ), '3.9.1', '', 'This filter is no longer necessary.' );
 		$files            = array();
 		$success          = 0;
 		$error            = 0;
 		$output           = array();
-
+		
 		foreach ( $paths_to_cleanup as $path ) {
-			if ( ! function_exists( 'list_files' ) ) {
-				include_once( ABSPATH.'wp-admin/includes/file.php' );
-			}
-			if ( $listed_files = list_files( $path, $folders_level ) ) {
-				$files = array_merge( $files, $listed_files );
+			if ( $wp_filesystem->is_dir( $path ) ) {
+				$listed_files = $wp_filesystem->dirlist( $path, true, true );
+				
+				if ( $listed_files ) {
+					foreach ( $listed_files as $fileinfo ) {
+						$files[] = trailingslashit( $path ) . $fileinfo['name'];
+					}
+				}
 			}
 		}
-
+		
 		if ( ! empty( $files ) ) {
```

Comment:
I got this message instead of "Nothing to delete":
![image](https://github.com/user-attachments/assets/db9ac07c-ba9f-4d0c-bb7d-99c72ec38233)


I registered a log to see files:
```php
["C:\\laragon\\www\\mohamad\/wp-content\/uploads\/wpo_wcpdf_5084691e48ba13ddf24dc1d1678402bd\/attachments\/.htaccess","C:\\laragon\\www\\mohamad\/wp-content\/uploads\/wpo_wcpdf_5084691e48ba13ddf24dc1d1678402bd\/attachments\/index.php","C:\\laragon\\www\\mohamad\/wp-content\/uploads\/wpo_wcpdf_5084691e48ba13ddf24dc1d1678402bd\/dompdf\/.htaccess","C:\\laragon\\www\\mohamad\/wp-content\/uploads\/wpo_wcpdf_5084691e48ba13ddf24dc1d1678402bd\/dompdf\/index.php","C:\\laragon\\www\\mohamad\/wp-content\/uploads\/wpo_wcpdf_5084691e48ba13ddf24dc1d1678402bd\/dompdf\/log.htm"]
```